pkg/metrics/metrics.jsx

```diff
@@ -663,10 +722,38 @@ class MetricsMinute extends React.Component {
         let events = null;
         if (this.props.events) {
             const timestamp = this.props.startTime + (this.props.minute * 60000);
-            events = <dl className="metrics-events" style={{ "--metrics-minute": this.props.minute }}>
-                <dt><time>{ moment(timestamp).format('LT') }</time></dt>
-                { this.props.events.map(t => <dd key={ t }>{ RESOURCES[t].event_description }</dd>) }
-            </dl>;
+            const desc = <div className="description">
+                { this.props.events.events.map(t => <span className="type" key={ t }>{ RESOURCES[t].event_description }</span>) }
+                <div className="time">
+                    <time>{ moment(timestamp).format('LT') }</time>
+                    {this.state.expanded && this.state.logsUrl ? <Button variant="link" isInline onClick={e => cockpit.jump(this.state.logsUrl)}>{_("All logs")}</Button> : null}
+                </div>
+            </div>;
+
+            let body = " "; // Cannot be false-y, otherwise table does not show '>'
+            if (this.state.expanded) {
+                body = <div className="cockpit-log-panel">
+                    {this.state.logs === null ? _("Loading...") : this.state.logs.length === 0 ? _("No logs found") : this.state.logs}
+                </div>;
+            }
+
+            const entry = [{
+                props: { key: timestamp, 'data-row-id': timestamp },
+                columns: [{ title: desc }],
+                hasPadding: false,
+                expandedContent: body,
+            }];
+
+            events = <ListingTable aria-label={ _("Event logs") }
+                                   className="metrics-events"
```

Comment:
I think we want to add gridBreakPoint='' here to avoid collapsing into the mobile mode:

Currently is looks like this:
![Screen Shot 2021-03-11 at 11 02 03](https://user-images.githubusercontent.com/14921356/110770318-79accc80-8259-11eb-9d6c-baeacf87f355.png)


Removing the breakpoint makes it like this:
![Screen Shot 2021-03-11 at 11 04 25](https://user-images.githubusercontent.com/14921356/110770386-8e896000-8259-11eb-9c4a-72c71b10bff7.png)

What do you think?

Reply:
awesome! I saw this before and was not sure what was happening and then forgot about it.
Happy to see that the fix is that simple!